js/helper.js

```diff
@@ -1,84 +1,90 @@
       }
 
-      var paramType = commandOptions[sourceMapOption].type;
-      var paramName = commandOptions[sourceMapOption].param;
-
-      if (paramType === 'array') {
-        if (!Array.isArray(paramValue)) {
-          throw new Error(sourceMapOption + ' should be an array');
-        }
-        return newOptions.concat(
-          transformOption(commandOptions[sourceMapOption], paramValue)
-        );
-      } else if (paramType === 'boolean' || paramType === 'inverted-boolean') {
-        if (typeof paramValue !== 'boolean') {
-          throw new Error(sourceMapOption + ' should be a bool');
-        }
-        if (paramType === 'boolean' && paramValue) {
-          return newOptions.concat([paramName]);
-        } else if (paramType === 'inverted-boolean' && paramValue === false) {
-          return newOptions.concat([paramName]);
-        }
-        return newOptions;
+      return newOptions.concat(
+        transformOption(commandOptions[sourceMapOption], paramValue)
+      );
+    }
+
+    if (paramType === 'boolean' || paramType === 'inverted-boolean') {
+      if (typeof paramValue !== 'boolean') {
+        throw new Error(`${sourceMapOption} should be a bool`);
       }
-      return newOptions.concat(paramName, paramValue);
-    }, []);
-  },
 
-  prepareCommand: function(command, commandOptions, options) {
-    return command.concat(this.normalizeOptions(commandOptions || {}, options || {}));
-  },
+      if (paramType === 'boolean' && paramValue) {
+        return newOptions.concat([paramName]);
+      }
+
+      if (paramType === 'inverted-boolean' && paramValue === false) {
+        return newOptions.concat([paramName]);
+      }
 
-  getPath: function() {
-    if (process.env.NODE_ENV === 'test') {
-      return path.resolve(__dirname, '__mocks__/sentry-cli');
+      return newOptions;
     }
-    return binaryPath;
-  },
-
-  execute: function(args) {
-    var env = Object.assign({}, process.env);
-    var that = this;
-    return new Promise(function(resolve, reject) {
-      childProcess.execFile(that.getPath(), args, { env: env }, function(err, stdout) {
-        if (err) {
-          reject(err);
-        } else {
-          resolve(stdout);
-        }
-      });
-    });
+
+    return newOptions.concat(paramName, paramValue);
+  }, []);
+}
+
+function prepareCommand(command, commandOptions, options) {
+  return command.concat(normalizeOptions(commandOptions || {}, options || {}));
+}
+
+function getPath() {
+  if (process.env.NODE_ENV === 'test') {
+    return path.resolve(__dirname, '__mocks__/sentry-cli');
   }
+
+  return binaryPath;
+}
+
+function execute(args) {
+  const env = Object.assign({}, process.env);
+  return new Promise((resolve, reject) => {
+    childProcess.execFile(getPath(), args, { env }, (err, stdout) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(stdout);
+      }
+    });
+  });
+}
+
+module.exports = {
+  normalizeOptions,
+  prepareCommand,
+  getPath,
```

Comment:
In `helper.js` all functions were moved to the top scope so that no more `this` is necessary.